.github/workflows/auto-pr.yml

```
@@ -0,0 +1,40 @@
+      - 'feature/**'
+
+jobs:
+  auto-create-pr:
+    runs-on: ubuntu-latest    
+    permissions:
+      contents: write
+      pull-requests: write
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)